### PR TITLE
Rename training images

### PR DESCRIPTION
**Pull request by attard-andrew**
I created a new folder called 'new_training_images' and move all available training images to here, renaming them by number: 0.jpg, 1.jpg, 2.jpg etc.

### CODE REVIEW

**Comment by ritwik12** [comment]
@attard-andrew We need all files in their individual folders only, with numbering 01, 02, 03 etc.
So for earth we will have 01, 02, 03 etc files and same for mars and others

**Comment by attard-andrew** [comment]
@ritwik12 - ok, will update. What's the usual process in this scenario? Close the pull request?

**Comment by ritwik12** [comment]
No, Just made changes to your fork and push them

**Comment by ritwik12** [comment]
@attard-andrew When you commit your new changes in your branch. It will automatically updated in the PR.

**Comment by attard-andrew** [comment]
@ritwik12 got it - thanks!

**Comment by attard-andrew** [comment]
@ritwik12 How's that? :)

**Comment by ritwik12** [comment]
@ attard-andrew Good try, but some file names are not changed and you should follow 001 concept instead of just 1 so that files will be ordered as per numeric value as many OS and GitHub don't follow natural sorting. so files will be 001,002,003,004 ..... 010......100 like that... and you've given numbers continuesly for all objects which we don't want. We need numbering per directory like for 25 files in earth it'll be 001-025 numbering and for Mars same as 001-025 not 026-050

**Comment by attard-andrew** [comment]
@ritwik12 I think (hope) I got it right this time!

**Comment by ritwik12** [comment]
@attard-andrew That's really great, thanks for your good work. Please check why there is a loss of images, I can see Neptune have only 16 images now and it was 25 previously.
Eg:  .../image_retraining/training_data/neptune/18_135484_Rendering-of-the-Gas-Planet-Neptune.jpg 
 hub/examples/image_retraining/training_data/earth/114_maxresdefault.jpg

**Comment by attard-andrew** [comment]
@ritwik12 - Unclear to me what caused it, but I re-copied the original image directory, re-ran the script, then compared file counts to check for discrepancies.  Have a look

File: utils/rename_files.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#! python3
+# rename_training_files.py - Will be used to rename all training data image files
```
Comment (ritwik12) [inline utils/rename_files.py]:
```suggestion
# rename_files.py - Will be used to rename all training data image files
```

**Comment by ritwik12** [comment]
@attard-andrew Please change name as suggested

**Comment by ritwik12** [comment]
@attard-andrew Thanks a lot for your efforts :)

**Comment by attard-andrew** [comment]
@ritwik12 Thanks for your patience! This was pretty exciting for me, even if just to go through the workflow for the first time :)

**Comment by ritwik12** [comment]
Sure, Always welcome to contribute here and other repositories as well. @attard-andrew If you need any help feel free to ping

**Comment by attard-andrew** [comment]
> Sure, Always welcome to contribute here and other repositories as well. @attard-andrew If you need any help feel free to ping

Thanks!

**Pull request merged**